src/lib/datepicker/calendar.ts

```diff
@@ -208,29 +230,37 @@ export class MatCalendar<D> implements AfterContentInit, OnDestroy, OnChanges {
     this._userSelection.emit();
   }
 
+  /** Handles month selection in the multi-year view. */
+  _yearSelected(year: D): void {
```

Comment:
`_selectYear`?

Reply:
I combined this with `_monthSelected` and named the new method `_goToDateInView`